### PR TITLE
Adding various artifacts to k8s cluster-report

### PR DESCRIPTION
**Pull request by gaurav-dalvi**
Resolves: https://github.com/noironetworks/support/issues/789

### CODE REVIEW

File: cmd/acikubectl/cmd/debug.go [hunk 1]
```diff
@@ -190,36 +227,37 @@ func clusterReport(cmd *cobra.Command, args []string) {
...
-			continue
-		}
+	// Execute kubectl commands
+	for _, cmd := range cmds {
+		go func(cmd reportCmdElem, tarWriter *tar.Writer){
+			buffer := new(bytes.Buffer)
+			now := time.Now()
+			defer wg.Done()
+
+			fmt.Fprintln(os.Stderr, "Running command: kubectl", strings.Join(cmd.args, " "))
```
Comment (puppet-py) [inline cmd/acikubectl/cmd/debug.go]: Request: <srings> pkg seems to be getting used only in this "one line (Join)" in the entire file.  Can we avoid that?
Possibly if we construct the args with single element then this join can be avoided, somewhat like this:
args: []string{"describe nodes -n " + systemNamespace}
args: []string{"describe daemonset aci-containers-host -n " + systemNamespace}

Comment (puppet-py) [reply cmd/acikubectl/cmd/debug.go]: I meant the "strings" pkg

File: cmd/acikubectl/cmd/debug.go [hunk 2]
```diff
@@ -87,6 +89,41 @@ func clusterReport(cmd *cobra.Command, args []string) {
 			name: "cluster-report/logs/controller/acc.log",
 			args: accLogCmdArgs(systemNamespace),
 		},
+		{
+			name: "cluster-report/status/describe_nodes.log",
```
Comment (puppet-py) [inline cmd/acikubectl/cmd/debug.go]: Request: if can keep consistency in the name of the files. For "all describe resources" of k8s we are creating <resource>_status.log, can we do the same for the "describe nodes" ?

File: cmd/acikubectl/cmd/debug.go [hunk 3]
```diff
@@ -190,36 +227,37 @@ func clusterReport(cmd *cobra.Command, args []string) {
...
+	for _, cmd := range cmds {
+		go func(cmd reportCmdElem, tarWriter *tar.Writer){
+			buffer := new(bytes.Buffer)
+			now := time.Now()
+			defer wg.Done()
+
+			fmt.Fprintln(os.Stderr, "Running command: kubectl", strings.Join(cmd.args, " "))
+			err := execKubectl(cmd.args, buffer)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, err)
```
Comment (snaiksat) [inline cmd/acikubectl/cmd/debug.go]: Don't you need to bail out here?

Comment (gaurav-dalvi) [reply cmd/acikubectl/cmd/debug.go]: Thanks for catching. Done.

**Pull request merged**